lib/sandbox/postman-legacy-interface.js

```diff
@@ -351,5 +345,29 @@ module.exports = {
         }
 
         scope.__postman_legacy_setup = false;
+    },
+
+    /**
+     *
+     * @param  {Execution} execution
+     *
+     * @todo Move getAssertionObject function from pmapi-setup-runner.js to a common place
+     * and reuse it here
+     */
+    raiseAssertionEvents: function (execution, pmapi, tests) {
+        var assertionIndex = pmapi.test.index(),
+            assertionEventName = EXECUTION_ASSERTION_EVENT_BASE + execution.id,
+            assertions = _.map(tests, function (value, key) {
+                return {
+                    name: String(key),
+                    skipped: false,
+                    passed: Boolean(value),
```

Comment:
Casting here so that later we don't have to iterate over tests and change `undefined` to `false`.
I think it's safe as far as both of these are treated equally
```js
tests['foo'] = true;
tests['foo'] = 'some message that will never be shown to user';
```
@kamalaknn Though I have tested this in App, pointing this out for a second check.